Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/CreateAndQueryChangeRequestV2/task.json

```diff
@@ -14,7 +14,7 @@
   ],
```

Comment:
Is this visibility and runsOn correct for Pipelines/Checks  ?

Reply:
Yes "ServerGate" works in checks